src/app/core/import/import-column-mapping/import-column-mapping.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from "@angular/core";
+import { ColumnMapping } from "../column-mapping";
+import { EntitySchema } from "app/core/entity/schema/entity-schema";
+
+@Injectable({
+  providedIn: "root",
+})
+export class ImportColumnMappingService {
```

Comment:
I am adding a short comment (JsDoc), so that other developers quickly understand the purpose of this service. That comment is also displayed by IDEs as a tooltip, when written in the `/**` style.

```suggestion
/**
 * Infer most likely entity fields that the column from an imported file could be mapped to.
 * This is used by the Import Module to give users smart pre-selections in the user interface.
 */
@Injectable({
  providedIn: "root",
})
export class ImportColumnMappingService {
```